src/npc.cpp

```diff
@@ -2175,6 +2175,14 @@ void npc::shop_restock()
     distribute_items_to_npc_zones( ret, *this );
 }
 
+std::string npc::get_interval() const
```

Comment:
Hi can you change the function name to get_restock_interval()

Reply:
Also very cool change can't wait for it to get merged!

Reply:
aye thanks, tending to it rn